ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature

```diff
@@ -0,0 +1,112 @@
+# @sustainer: acadova@redhat.com
+
+@ui
+@api-provider
+@api-connector
+@import
+@export
+
+Feature: API Provider Integration - Import Export
+
+  # TODO: import/export tests with OpenAPI resource referenced from api-provider-based integration
+
+  Background:
+    Given log into the Syndesis
+    And clean application state
+    And truncate "todo" table
+    And Set Todo app credentials
+
+  @import-export-open-api
+
+  Scenario: Create an integration with custom API connector and then export and import the integration
+
+    When click on the "Customizations" link
+    And navigate to the "API Client Connectors" page
+    And click on the "Create API Connector" link
+    And check visibility of page "Upload Swagger Specification"
+    Then upload swagger file
+      | file | swagger/connectors/todo.swagger.yaml |
+
+    When click on the "Next" button
+    Then check visibility of page "Review Actions"
+
+    When click on the "Next" link
+    Then check visibility of page "Specify Security"
+
+    And click on the "Next" button
+    And fill in values by element data-testid
+      | name     | Todo connector |
+      | basepath | /api           |
+    And fill in TODO API host URL
+    And click on the "Save" button
+
+    When created connections
+      | Todo connector | todo | Todo connection | no validation |
+    And navigate to the "Home" page
+
+    When create an API Provider integration "TODO Integration get single" from file swagger/connectors/todo.json
+    And select API Provider operation flow Fetch task
+    Then check flow title is "Fetch task"
```

Comment:
Better for what you are doing would be Create new task, as you are creating new tasks in `Fetch task`